### PR TITLE
Adds formatting support for Kibana APIs

### PR DESCRIPTION
**Pull request by tylersmalley**
This was added in a similar format to the already existing code, though I did some minor cleaning up to better understand it. It still references Sense, which I am not sure if we can remove that. If we no longer build docs referencing Sense examples, then we should.

This code could really benefit from a more thorough cleanup or possible rewrite in the future.

screenshot 2018-02-06 07 48 13

screenshot 2018-02-06 07 48 29

Also fixed Copy as cURL commands to include the hostname configured through the settings.

### CODE REVIEW

**Comment by tylersmalley** [comment]
This is a work in progress until I get the corresponding Kibana PR up to provide complete testing.

**Comment by tylersmalley** [comment]
I need to revert the template literals change here.

**Comment by tylersmalley** [comment]
Reverted template literals. Working on getting API PR up on Kibana.

File: resources/web/docs.js [hunk 1]
```diff
@@ -303,8 +317,14 @@ jQuery(function() {
           curlText += comment + '\n';
         } else {
           path = path.replace(/^\//, '').replace(/\s+$/,'');
-          path += path.includes('?') ? '&pretty' : '?pretty';
-          curlText += 'curl -X' + method + " 'localhost:9200/" + path + "'";
```
Comment (tylersmalley) [inline resources/web/docs.js]: `localhost:9200` was hardcoded here. I updated it to include the host/port configured through the settings.

**Comment by debadair** [comment]
LGTM

We do still publish docs that reference Sense, so for the time being that's still valid.

**Pull request merged**